interview-questions.md

```diff
@@ -8,62 +8,61 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
```

Comment:
All the answers are good. When interviewing you want the company to get to know you better so share your experience with each conceptual question.